go/mysql/conn.go

```diff
@@ -273,10 +275,10 @@ func (c *Conn) readHeaderFrom(r io.Reader) (int, error) {
 		return 0, vterrors.Wrapf(err, "io.ReadFull(header size) failed")
 	}
 
-	sequence := uint8(header[3])
-	if sequence != c.sequence {
-		return 0, vterrors.Errorf(vtrpc.Code_INTERNAL, "invalid sequence, expected %v got %v", c.sequence, sequence)
-	}
+	//sequence := uint8(header[3])
```

Comment:
What's this about?

Reply:
Will remove this edit. Was having trouble with code earlier in the process...